include/matx/operators/clone.h

```diff
@@ -116,19 +116,20 @@ namespace matx
    * @return operator to compute the cloned value
    */
   template <int Rank, typename Op>
-    auto __MATX_INLINE__ clone(Op t, const index_t (&shape)[Rank])
+    auto __MATX_INLINE__ clone(Op t, const std::array<index_t, Rank> shape)
```

Comment:
reference?

Reply:
no,  i'm moving away from any operators using reference since it prevents inline usage  i.e.  clone(someotherop(asdf));

Reply:
fixed